py/torch_tensorrt/dynamo/backend/lowering/_partition.py

```diff
@@ -55,6 +55,10 @@ def __init__(
         )
 
         self.min_block_size = min_block_size
+        logger.debug(
+            "Initialized Capability-Based Partitioner with available Converters:\n"
+            + f"{CONVERTERS.display_all_available_converters()}"
+        )
```

Comment:
Added logging and debug functionality to print all available converters during partitioning.

Reply:
This doesnt need to be printed every time a partitioner is created, IMO the best place is to actually print a debug message every time a converter is registered so at runtime as new ones get added it will get logged (so probs part of the converter decorator or whenever it gets added to the dictionary - imo should be fine to add this to fx.trt_converter as well as it is just a logging change